.tugboat/config.yml

```diff
@@ -0,0 +1,122 @@
+services:
+  # What to call the service hosting the site.
+  php:
+    # Match this with the dockerfile in the lagoon folder.
+    image: tugboatqa/php:8.1-fpm
```

Comment:
I tried using the actual docker image as specified in the `lagoon` directory but they require a Volume, which Tugboat doesn't support, so I tried to match it as closely as I could to `tugboatqa` images.